TbsCore/Helpers/AccountHelper.cs

```diff
@@ -68,6 +69,13 @@ public static void StartAccountTasks(Account acc)
                 TaskExecutor.AddTaskIfNotExists(acc, new GetTribe() { ExecuteAt = DateTime.MinValue.AddHours(3) });
             }
 
+            // check server version

```

Comment:
I'm not entirely sure if you can check server info/sitter account here since you can't be certain that bot is already logged into the account. Move this logic elsewhere - maybe after login task, where you also check if account was successfully logegd in?